public/stylesheets/style.less

```diff
@@ -2012,7 +2012,7 @@ button.return-btn {
 // Basic Page Layout
 
 .page-wrapper {
-  top: 0px;
```

Comment:
Maybe put all of the `top` edits you made into some classes. Then you can easily turn the banner on and off.